lib/pbench/server/api/resources/datasets_list.py

```diff
@@ -45,24 +45,30 @@ def __init__(self, config: PbenchServerConfig, logger: logging.Logger):
             role=API_OPERATION.READ,
         )
 
-    def _get(self, json_data: JSON, request: Request) -> Response:
+    def _get(self, _, request: Request) -> Response:
         """
         Get a list of datasets matching a set of criteria.
 
         NOTE: This does not rely on a JSON payload to identify the dataset and
         desired metadata keys; instead we rely on URI query parameters.
 
+        Args:
+            json_data: For a GET, this contains only the Flask URI template
```

Comment:
Why are we documenting `json_data` here, but have a `_` in the function definition?